libs/model/src/main/kotlin/de/cyface/model/RequestMetaData.kt

```diff
@@ -60,7 +59,8 @@ data class RequestMetaData(
     val logCount: Int,
     val imageCount: Int,
     val videoCount: Int,
-    val filesSize: Long
+    val filesSize: Long,
+    val attachmentIdentifier: String? = null,
```

Comment:
That's a code smell. There should be a different way to handle this, with either delegation or inheritance.